website/docs/10_going-deep/instrumentation.md

```diff
@@ -0,0 +1,138 @@
+# Booster instrumentation
+
+## Trace Decorator
+The Trace Decorator is a **Booster** functionality that facilitates the reception of notifications whenever significant events occur in Booster's core, such as event dispatching or migration execution.
+
+### Usage
+To configure a custom tracer, you need to define an object with two methods: onStart and onEnd. The onStart method is called before the traced method is invoked, and the onEnd method is called after the method completes. Both methods receive a TraceInfo object, which contains information about the traced method and its arguments.
+
+Here's an example of a custom tracer that logs trace events to the console:
+
+```typescript
+import {
+  TraceParameters,
+  BoosterConfig,
+  TraceActionTypes,
+} from '@boostercloud/framework-types'
+
+class MyTracer {
+  onStart(config: BoosterConfig, actionType: TraceActionTypes, traceParameters: TraceParameters): void {
+    console.log(`Start ${TraceActionTypes[actionType]}: ${traceParameters.className}.${traceParameters.methodName}`)
+  }
+
+  onEnd(config: BoosterConfig, actionType: TraceActionTypes, traceParameters: TraceParameters): void {
+    console.log(`End ${TraceActionTypes[actionType]}: ${traceParameters.className}.${traceParameters.methodName}`)
+  }
+}
+```
+
+You can then configure the tracer in your Booster application's configuration:
+
+```typescript
+import { BoosterConfig } from '@boostercloud/framework-types'
+import { MyTracer } from './my-tracer'
+
+const myTracer = new MyTracer()
+const config: BoosterConfig = {
+// ...other configuration options...
+  trace: {
+    enableTraceNotification: true,
+    verbose: false,
```

Comment:
Verbose? I think this is not in the configuration

Reply:
it is `includeInternal`. I renamed it and forgot the docs